functorch/_src/aot_autograd.py

```diff
@@ -466,11 +466,19 @@ def create_aot_dispatcher_function(
     # NB: don't bother setting allow_fallback_kernels; this should not actually
     # be configurable in fake tensor, we should automatically do the right
     # thing
+    if config.debug_fake_cross_ref:
+        # This is a little messy but TorchDynamo directly changes `use_fake_tensor`
+        # so it's not enough for user to change the config manually
+        # TODO: have TorchDynamo read in `use_fake_tensor` from os environ /
+        # coordinate flags
+        config.use_fake_tensor = False
```

Comment:
This doesn't look kosher. Why not use the property trick to make use_fake_tensor a computed value?

Reply:
What is the property trick ?

Reply:
check the bottom of torchdynamo/config.py. you'd customize setattr